server/auth/permission.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissionKeys and
+// limitations under the License.
+//
+
+package auth
+
+import (
+	"strings"
+
+	"github.com/tikv/pd/pkg/errs"
+)
+
+// Action represents rbac actions.
+type Action string
+
+// All available actions types.
+const (
```

Comment:
How about using the way like operator kind?

Reply:
We are assuming that an Action can hold only one operation now (so that AddPermission/RemovePermission can be simple and direct).
Maybe we won't gain much from changing those consts to uints?

Reply:
Hmmm...I'm not sure if it is reasonable.